build.gradle

```diff
@@ -35,12 +35,12 @@ buildscript {
         guavaVersion = '20.0'
         protobufGradlePluginVerison = '0.8.0'
         
-        spineVersion = '0.9.14-SNAPSHOT'
+        spineVersion = '0.9.14.1-SNAPSHOT'
         
         //TODO:2016-12-12:alexander.yevsyukov: Advance the plug-in version together with all
         // the components and change the version of the dependency below to
         // `spineVersion` defined above.
-        spineToolsVersion = '0.9.5-SNAPSHOT'
+        spineToolsVersion = '0.9.6-SNAPSHOT'
```

Comment:
@alexander-aleksandrov FYI. I have updated the version already. See my recent commits to this branch.